api/migrations/0030_auto_20180426_1313.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.0.1 on 2018-04-26 13:13
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0029_auto_20180426_0945'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AllocationsHistory',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+            ],
+        ),
+        migrations.RemoveField(
+            model_name='asset',
+            name='assigned_to',
+        ),
+        migrations.AddField(
+            model_name='asset',
+            name='current_owner',
+            field=models.CharField(default=django.utils.timezone.now, editable=False, max_length=50),
```

Comment:
please change this default.. date isn't valid for an integer field

Reply:
Done.